src/pages/dho/Explore.vue

```diff
@@ -186,13 +211,18 @@ q-page.page-explore
           q-btn.q-px-lg.h-btn1(no-caps rounded unelevated color="secondary" href="https://hypha.earth/" target="_blank") Discover More
 
   .row.q-py-md
-    .col-sm-12.col-md-12.col-lg-9(ref="scrollContainer")
+    .col-sm-12.col-md-12.col-lg-9(ref="scrollContainer" v-if="exploreBy === 'DAOS'")
```

Comment:
@arsenijesavic It is still using strings here ( `DAOS` ) please check everywhere